go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/alecthomas/kingpin v2.2.6+incompatible
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
 	github.com/bitly/go-simplejson v0.5.0
+	github.com/cenkalti/backoff/v4 v4.1.2
```

Comment:
IMHO backoff does not need a dependency...

Reply:
We already have our bicycle: file.d. Let's avoid creating other one for exponential backoff.

Reply:
Sure that is up to you, it is not blocking
I just think that  a new dependency for `t = t * 1.2` is too much